iptstate.cc

```diff
@@ -2958,6 +2969,7 @@ int main(int argc, char *argv[])
   } // end while(1)
 
   out:
+    cleanup(stable);
```

Comment:
unindent to match the rest of the function please.

Reply:
Right, fixed.